src/Integration/RequestIntegration.php

```diff
@@ -23,7 +23,7 @@
      */
     public function setupOnce(): void
     {
-        Scope::addGlobalEventProcessor(function (Event $event) {
+        Scope::addGlobalEventProcessor(function (Event $event): Event {
```

Comment:
nit: Not really relevant but an `EventProcessor` could also return `null` so while in this case this will not happen it is possible for an `EventProcessor` to discard an `Event` by returning `null`.

Reply:
Indeed, but the current function can't return `null` so it makes no sense to typehint the return type against a `nullable` value